homeassistant/components/nanoleaf/device_trigger.py

```diff
@@ -60,6 +61,15 @@ async def async_attach_trigger(
     trigger_info: TriggerInfo,
 ) -> CALLBACK_TYPE:
     """Attach a trigger."""
+    async_create_issue(
```

Comment:
This issue is only created once it triggers, therefore it should set the `is_persistent` flag to ensure it keep here after restarts.

Reply:
It doesn't create when triggered, it is created when this is used. I even tested this out :)

Reply:
oh right 🤦 **attach**_trigger 

<img src="https://media0.giphy.com/media/xUOrw6zv9v5gr9z6X6/giphy.gif"/>